### PR TITLE
Tickets/dm 24309

### PR DESCRIPTION
**Pull request by patrickingraham**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -4,54 +4,146 @@
   See https://developer.lsst.io/restructuredtext/style.html
   for a guide to reStructuredText writing.
```
Comment (tribeiro) [inline index.rst:5]: Are you going to add the `note` with an abstract like text?

File: index.rst [hunk 2]
```diff
@@ -4,54 +4,146 @@
...
+| Pixel pitch                        |  10um             |
++------------------------------------+-------------------+
+| Plate Scale                        |  100 um/arcsec    |
+|                                    |  0.1 arcsec/pixel |
++------------------------------------+-------------------+
+| Secondary Obscuration Diameter     |  0.423m           |
++------------------------------------+-------------------+
+
+The optical design (in Zemax) for the telescope and LATISS instrument can be found in
+`Document-27702 <https://docushare.lsst.org/docushare/dsweb/Get/Document-27702>`_ (credentials required). The
```
Comment (tribeiro) [inline index.rst]: Double "The". One at the end of this line and one at the beginning of the next line.

File: index.rst [hunk 3]
```diff
@@ -4,54 +4,146 @@
...
+
+The CWFS technique for collimation (and focus) is highly affected by seeing, particularly dome seeing. Experience
+has shown that long (~30s) exposures are required to average out the effects of dome seeing to have a stable solution.
+Seeing variations of ~20nm for the defocus and coma Zernikes is to be expected.
+
+The telescope has very good optics (0.06 waves RMS) and therefore image quality is fully seeing limited. The CWFS
+method measures the wavefront error (WFE) introduced by the optics and their alignment beyond the seeing limit
+therefore so long as the
+total WFE is at (or below) the diffraction limit of the system then optimizing beyond this point is of no use. Using
+a diffraction limit of lambda/4 for 632nm gives 158nm. Assuming all the WFE comes from the defocus through spherical
```
Comment (tribeiro) [inline index.rst]: Can you make these equations? it would make it easier to read it it.

File: index.rst [hunk 4]
```diff
@@ -4,54 +4,146 @@
...
+aberration terms (4-11), then this allows each term to have a magnitude of $158/sqrt(8)$~55nm. Therefore, we
```
Comment (tribeiro) [inline index.rst]: Check https://developer.lsst.io/restructuredtext/style.html#inline-math for how to make inline equations.

```
$158/sqrt(8)$~55nm
```

would probably be something line

```
:math:`158 / sqrt(8) \sim 55\mathrm{nm}`
```

File: index.rst [hunk 5]
```diff
@@ -4,54 +4,146 @@
...
+consider any focus or coma abberation magnitude below 50nm to be acceptable and not require further optimization as
+the PSF will be heavily seeing dominated.
+
+One should note that the above technique is only an approximation and breaks down in certain cases. For example, if one
+has 158nm of coma and all over aberrations are zero then you will see very significant coma in the PSF! The ~50 nm
```
Comment (tribeiro) [inline index.rst]: `and all over aberrations` -> `and all other aberrations` ?

File: index.rst [hunk 6]
```diff
@@ -4,54 +4,146 @@
...
+The telescope has very good optics (0.06 waves RMS) and therefore image quality is fully seeing limited. The CWFS
+method measures the wavefront error (WFE) introduced by the optics and their alignment beyond the seeing limit
+therefore so long as the
+total WFE is at (or below) the diffraction limit of the system then optimizing beyond this point is of no use. Using
+a diffraction limit of lambda/4 for 632nm gives 158nm. Assuming all the WFE comes from the defocus through spherical
+aberration terms (4-11), then this allows each term to have a magnitude of $158/sqrt(8)$~55nm. Therefore, we
+consider any focus or coma abberation magnitude below 50nm to be acceptable and not require further optimization as
+the PSF will be heavily seeing dominated.
+
+One should note that the above technique is only an approximation and breaks down in certain cases. For example, if one
```
Comment (tribeiro) [inline index.rst]: Not sure I understood exactly what you are trying to convey in this paragraph. 

It starts by saying that this is an approximation that breaks down in certain cases, but then example doesn't seem to suggest a case where the approximation actually fails. Unless you wanted to say

> For example, if one has 158nm of coma **(equivalent to ~50nm of decenter),** and all ~over~ **other** aberrations are zero**,** then you ~will~ **would not** see ~very~ significant coma in the PSF!

File: index.rst [hunk 7]
```diff
@@ -4,54 +4,146 @@
...
+Due to the above discussion, we encourage that **no corrections be applied for motions below 2um in Z and 50 um in X
+and Y.**
+These values correspond to ~10nm of WFE. This is done in order to reduce unnecessary wear on the hexapod and cause
+unnecessary heating.
+
+Another aspect of the CWFS solution which is not yet well understood is strong variations in astigmatism measurements.
+The only mechanism to have any control on astigmatism is to adjust the pressure under the mirror, however testing has
+shown that small corrections are not easily achieved. In the figure below there is an example of a large coefficient
+term for astigmatism. These are believed to be fitting issues as they can vary strongly from image to image. At the
+time of writing this issue is still be tracked down.
```
Comment (tribeiro) [inline index.rst]: Can you rephrase this?

> At the time of writing this issue is still be tracked down.

It reads a bit odd to me. I think it should be something like:

> At the time of this writing, this issue is still being tracked down.

or 

> At the time of this writing, we have not found the source of this issue.

File: index.rst [hunk 8]
```diff
@@ -4,54 +4,146 @@
...
+
+
+.. figure:: /_static/cwfs_fit_example.jpg
```
Comment (tribeiro) [inline index.rst:90]: I don't think figure 1 is ever mentioned in the text. I think it would be good to add a reference to it, explaining why it is here.

File: index.rst [hunk 9]
```diff
@@ -4,54 +4,146 @@
...
+
+One should note that the above technique is only an approximation and breaks down in certain cases. For example, if one
+has 158nm of coma and all over aberrations are zero then you will see very significant coma in the PSF! The ~50 nm
+guideline is meant as a guideline and is not a hard truth. In exceptional seeing conditions (which we've not
+encountered to date), the instrumental PSF may begin to have an effect. To date, the PSF quality is dominated by dome
+seeing and tracking error.
+
+In correcting the collimation/focus using the hexapod it is important to understand the limits of motion. Motion of the
+hexapod and how it relates to WFE correction is defined by a sensitivity matrix. The derivation of this matrix is
+discussed in detail in `this technote <tstn-016.lsst.io>`_. For the purposes of this document, a user must know how the
```
Comment (tribeiro) [inline index.rst]: I think that "in this tech note" leaves the impression that it is this tech note. Maybe rephrase to something like; 

> The derivation of this matrix is discussed in detail in `tstn-016 <https://tstn-016.lsst.io>`_.

File: index.rst [hunk 10]
```diff
@@ -4,54 +4,146 @@
...
+    :name: fig-fit-example
+
+    This shows what is believed to be a false measurement in astigmatism, which will often vary significantly
+    from image to image. In this case, only a focus offset should be applied as the coma values are acceptable.
+
+LatissCWFSAlign script
+======================
+
+To collimate and focus the Auxiliary telescope, the scriptQueue script
+`LatissCWFSAlign <https://github.com/lsst-ts/ts_externalscripts/blob/develop/python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py>_`
```
Comment (tribeiro) [inline index.rst]: This link is malformed in the technote. The tailing "_" if before the "`" should be:

```
`LatissCWFSAlign <https://github.com/lsst-ts/ts_externalscripts/blob/develop/python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py>`_
```

File: index.rst [hunk 11]
```diff
@@ -4,54 +4,146 @@
...
+is used. The script does not require the scriptQueue and is most easily run from a notebook. The script proceeds
```
Comment (tribeiro) [inline index.rst]: I think this is actually true for all `Scripts`... Maybe say:

> The script can be easily run from a notebook, which allows users to exercise a finer control over the process, while it is still not fully automated. The script proceeds

File: index.rst [hunk 12]
```diff
@@ -4,54 +4,146 @@
...
+from a "best focus" position, which can be a rough guess is required, then pistons the hexapod by a set amount,
+generally +/-0.8mm, which corresponds to +/-32.8 mm at the focal plane. Testing has shown this to be an adequate amount,
+balancing the amount of data required to calculate
+the low order Zernikes with keeping the image cutout size small to reduce the overhead of calculating the result.
+
+The CWFS calculation is currently performed using the package developed by Bo Xin, and is currently located
+`here <https://github.com/bxin/cwfs>`_. Note that this package (and our script) outputs Annular Zernike Noll
```
Comment (tribeiro) [inline index.rst]: I think you need to add double "_" when you do an inline link like this. A single "_" creates a reference that can be called later. 

https://developer.lsst.io/restructuredtext/style.html#external-links

File: index.rst [hunk 13]
```diff
@@ -4,54 +4,146 @@
...
+Polynomials. Because the AuxTel system only has 3 degrees of freedom adjustable via the hexapod (decenter X,Y and
```
Comment (tribeiro) [inline index.rst]: We actually have more than 3 degrees of freedom, we just decided not to exercise them, right? If we wanted to, we could rotate M2 as well. Maybe rephrase?

File: metadata.yaml [hunk 14]
```diff
@@ -17,6 +17,7 @@ doc_title: "Auxiliary Telescope: Using CWFS during operations and for collimatio
 
 # Author names, ordered as a list. Each author name should be formatted as 'First Last'
 authors:
+  - "Patrick Ingraham"
   - "Tiago Ribeiro"
```
Comment (tribeiro) [inline metadata.yaml:21]: Not sure I can be considered an author of this tech note in particular. Feel free to remove my name if you like.

**Review by tribeiro — APPROVED** [review]
I made a few comments here and there, nothing major. I would recommend at least taking a look at some of the formatting errors. Otherwise, good to go.

File: index.rst (hunk 7 above)
Comment (patrickingraham) [reply index.rst]: definitely forgot a comma. "At the time of writing, ...." is something you'll see, but it's not used much and a bit informal. You'll see it mostly in old newspapers.
I changed it to "These issues are still being tracked down"

File: index.rst (hunk 8 above)
Comment (patrickingraham) [reply index.rst:90]: it was, but wasn't linked. Corrected.

File: index.rst (hunk 12 above)
Comment (patrickingraham) [reply index.rst]: all links modified to have a double-underscore at the end (not double quotes), per your link.

File: metadata.yaml (hunk 14 above)
Comment (patrickingraham) [reply metadata.yaml:21]: No. Authorship doesn't mean you have to write words!
